hooks/pre-commit

```diff
@@ -3,14 +3,23 @@
 jsfiles=$(git diff --cached --name-only --diff-filter=ACM "*.js" "*.jsx" "*.ts" "*.tsx" | tr '\n' ' ')
 [ -z "$jsfiles" ] && exit 0
 
-# Check if any of the files are ugly
+# Check if any of the files are ugly or contain a console.log call
 uglyFiles=()
+consoleFiles=""
 for jsfile in $jsfiles; do
     diff "$jsfile" <($(npm bin)/prettier "$jsfile") >/dev/null || uglyFiles+=("$jsfile")
+    if [ "$(git diff --cached "$jsfile" | grep '^+.*console.log' -c)" -gt '0' ] ; then
+        consoleFiles="$consoleFiles '$jsfile'"
```

Comment:
Probably shouldn't use two different list formats in the same file. I don't mind which we use so long as it's consistent.

e.g.

```
consoleFiles=()
consoleFiles+("$jsfile")
```

Reply:
I fixed this and added npm run {lint,pretty}.

Sorry about the initial fault. Bash is hard.

Reply:
Oh, I was about to do this. You made the whole thing way prettier than I would have though so I guess it's better this way :)

There's still the issue of files with space in their names though. I'm not sure how to fix this now though. Maybe remove [`tr`](https://github.com/cmcaine/tridactyl/blob/bf03e7ed6ccafe2edcc7459d4c2f70a8f7cfe6d3/scripts/common#L4) and add `IFS='\n'` everywhere?